extensions/amp-animation/0.1/web-animations.js

```diff
@@ -441,20 +454,37 @@ export class MeasureScanner extends Scanner {
   }
 
   /**
-   * @param {string|!Element} targetSpec
-   * @return {!Element}
+   * @param {!WebKeyframeAnimationDef} spec
+   * @return {!Array<!Element>}
    * @private
    */
-  resolveTarget_(targetSpec) {
-    const target = user().assertElement(
-        typeof targetSpec == 'string' ?
-            this.context_.resolveTarget(targetSpec) :
-            targetSpec,
-        `Target not found: "${targetSpec}"`);
-    if (!this.targets_.includes(target)) {
-      this.targets_.push(target);
+  resolveTargets_(spec) {
+    let targets;
```

Comment:
I suggest failing if  both `selector` and `string target` are specified at the same time. Currently `selector` silently overrides `target` without user seeing the deprecated error.